scipy/stats/contingency.py

```diff
@@ -273,3 +275,124 @@ def chi2_contingency(observed, correction=True, lambda_=None):
+
+    Returns
+    -------
+    value : float
+        Value of the test statistic
+
+    References
+    ----------
+    .. [1] "Tschuprow's T",
+           https://en.wikipedia.org/wiki/Tschuprow's_T
+    .. [2] Bergsma, Wicher, "A bias-correction for Cramer's V and Tschuprow's T",
+           London School of Econ. and Pol. Sci., pp. 5-7.
+           http://stats.lse.ac.uk/bergsma/pdf/cramerV3.pdf
+    .. [3] Tschuprow, A. A. (1939) Principles of the Mathematical Theory of Correlation;
+           translated by M. Kantorowitsch. W. Hodge & Co.
+    .. [4] "Cramer's V", https://en.wikipedia.org/wiki/Cramer's_V
+    .. [5] "Nominal Association: Phi and Cramer's V",
+           http://www.people.vcu.edu/~pdattalo/702SuppRead/MeasAssoc/NominalAssoc.html
+    .. [6] Gingrich, Paul, "Association Between Variables", http://uregina.ca/~gingrich/ch11a.pdf
+
+
+    Examples
+    --------
+
+    2-way Example
+
+    >>> from scipy.stats.contingency import association, chi2_contingency
+    >>> obs = np.array([[100, 150], [203, 322], [42, 7], [32, 21]])
+    >>> num_obs = sum(obs.flatten())
+    >>> num_rows = obs.shape[0]
+    >>> num_cols = obs.shape[1]
+    >>> chi2 = tuple(chi2_contingency(obs))[0]
+
+    Pearson's contingency coefficient
+    >>> association(num_obs, num_rows, num_cols, chi2, stat="C")
+    0.22768
+
+    Cramer's V
+    >>> association(num_obs, num_rows, num_cols, chi2, stat="V")
+    0.23382
+
+    Tschuprow's T
+    >>> association(num_obs, num_rows, num_cols, chi2, stat="T")
+    0.17766
+
+    Phi
+    >>> association(num_obs, num_rows, num_cols, chi2, stat="phi")
+    0.23382
+
+
+    Notes
+    ------
+    Cramer's V and Tschuprow's T measure degree to which two variables are related, or the level of
+    their association. This differs from correlation, although many often mistakenly consider them equivalent.
+    Correlation measures in what way two variables are related, whereas, association measures
+    how related the variables are. As such, association does not subsume independent variables, and is
+    rather a test of independence. Where a value of 1.0 = perfect association or dependent variables, and
+    0.0 = no association or entirely independent variables.
+
+    Both the Cramer's V and Tschuprow's T are extensions of the phi coefficient. Moreover, due
+    to the close relationship between the Cramer's V and Tschuprow's T the returned values can often
+    be similar or even equivalent. They are likely to diverge more as the array shape diverges from a 2x2.
+
+    The evaluation of Pearsons Contingency Coefficient is not effected by the bias correction metric, because
+    it was not included as a part of the supporting academic paper.
+    """
+
+    try:
+        n_obs = int(n_obs)
+        n_cols = int(n_cols)
+        n_rows = int(n_rows)
+    except ValueError:
+        raise ValueError("n_obs, n_cols, n_rows must be int or float")
+
+    if stat.casefold() not in ['t', "c", 'v', 'phi']:
```

Comment:
can you use `lower` instead? it's clearer and completely identical here. also further down

Reply:
Made the change as you requested.